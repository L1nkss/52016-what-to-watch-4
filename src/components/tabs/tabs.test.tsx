import * as ShallowRenderer from 'react-test-renderer/shallow';
import Tabs from "@components/tabs/tabs";

const TABS = [`Overview`, `Details`, `Reviews`];

describe(`Testing Tabs component`, () => {
  it(`Component should successfully rendered`, () => {
    const myShallowRenderer = ShallowRenderer.createRenderer();
    myShallowRenderer.render(<Tabs activeTab={`Overview`} tabs={TABS} handleTabClick={() => {}} />);
    const result = myShallowRenderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});
