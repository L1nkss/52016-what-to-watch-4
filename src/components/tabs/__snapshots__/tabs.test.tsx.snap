// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Tabs component Component should successfully rendered 1`] = `
<nav
  className="movie-nav movie-card__nav"
>
  <ul
    className="movie-nav__list"
  >
    <WithActiveItem
      active="Overview"
      cb={[Function]}
      name="Overview"
    />
    <WithActiveItem
      active="Overview"
      cb={[Function]}
      name="Details"
    />
    <WithActiveItem
      active="Overview"
      cb={[Function]}
      name="Reviews"
    />
  </ul>
</nav>
`;
